src/raylib.cc

```diff
@@ -714,6 +714,11 @@ static PyObject* wait_for_next_message(PyObject* self, PyObject* args) {
   Worker* worker;
   PyObjectToWorker(worker_capsule, &worker);
   if (std::unique_ptr<WorkerMessage> message = worker->receive_next_message()) {
+    // The tuple constructed below will take ownership of a single None object.
+    // When the tuple goes out of scope, the reference count for None will be
+    // decremented. Therefore, we need to increment the reference count for None
+    // by one.
+    Py_INCREF(Py_None);
```

Comment:
I believe we actually need to do this once per use of the `None`, since each one is dereferenced separately.
You might do this by changing the code below to `... ? Py_INCREF(Py_None), Py_None : ...`.